lib/miq_pglogical/connection_handling.rb

```diff
@@ -2,7 +2,19 @@ class MiqPglogical
   module ConnectionHandling
     extend ActiveSupport::Concern
 
+    included do
+      delegate :logical_replication_supported?, :to => :class
+    end
+
     module ClassMethods
+      def logical_replication_supported?
+        return @logical_replication_supported if defined?(@logical_replication_supported)
+
+         is_superuser = ActiveRecord::Base.connection.exec_query("select usesuper from pg_user where usename = CURRENT_USER;", "SQL").first.first == true # first row, first column
```

Comment:
Minor, but capitalize the SQL keywords.
```suggestion
         is_superuser = ActiveRecord::Base.connection.exec_query("SELECT usesuper FROM pg_user WHERE usename = CURRENT_USER;", "SQL").first.first == true # first row, first column
```

Reply:
lol. jason pet peeve

Reply:
done.